<?xml version="1.0"?>
<launch>
  <arg name="pc_src" default="/filterd_cloud"/>
  <node pkg="Socket" type="server_test_point_cloud" name="server_test_point_cloud">
    <param name="pc_src" value="$(arg pc_src)"/>
  </node>
</launch>